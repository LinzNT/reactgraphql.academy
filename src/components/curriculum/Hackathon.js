import React from 'react'
import Ul, { Li } from '../layout/Ul'
import Session from './Session'

const Hackathon = ({ hideTitle }) => (
  <Session
    hideTitle={hideTitle}
    title="Last day real-world React challenge. We'll implement an app
    in teams from scratch"
  >
    <Ul>
      <Li>Create teams</Li>
      <Li>Discussion about architecture, features and tools</Li>
      <Li>Mini Hackathon! - work in teams to build apps</Li>
      <Li>
        Practice extreme programming and get support from the coaches and
        mentors
      </Li>
      <Li>Demos</Li>
    </Ul>
  </Session>
)

export default Hackathon
