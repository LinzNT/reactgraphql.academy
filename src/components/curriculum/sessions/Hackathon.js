import React from 'react'
import Ul, { Li } from '../../layout/Ul'
import Session from './Session'

const Hackathon = ({ title }) => (
  <Session title={title}>
    <Ul>
      <Li>GraphQL lecture</Li>
      <Li>Discussion about architecture, features and tools</Li>
      <Li>
        Practice extreme programming and get support from the coaches and
        mentors
      </Li>
      <Li>Trainees demos</Li>
    </Ul>
  </Session>
)

export default Hackathon
