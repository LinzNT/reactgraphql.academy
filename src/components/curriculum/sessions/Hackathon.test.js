import React from 'react'
import { shallow } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'

import Hackathon from './Hackathon'

describe('<Hackathon />', () => {
  it('should render component', () => {
    const wrapper = shallow(<Hackathon />)
    expect(shallowToJson(wrapper)).toMatchSnapshot()
  })
})
