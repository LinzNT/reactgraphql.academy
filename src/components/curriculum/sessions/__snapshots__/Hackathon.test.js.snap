// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Hackathon /> should render component 1`] = `
<Session>
  <styled.ul>
    <styled.li>
      Create teams
    </styled.li>
    <styled.li>
      Discussion about architecture, features and tools
    </styled.li>
    <styled.li>
      Hackathon! - work in teams to build apps
    </styled.li>
    <styled.li>
      Practice extreme programming and get support from the coaches and mentors
    </styled.li>
    <styled.li>
      Demos
    </styled.li>
  </styled.ul>
</Session>
`;
